### ISSUE TITLE
LD_SHIM_LIBS is not parsed as expected

### ISSUE THREAD

**Issue by IlyaBizyaev** [libhybris/libhybris#379]
https://github.com/libhybris/libhybris/blob/4a7d95b73bf40bf6d03d12bf270974c3d0278099/hybris/common/n/linker.cpp#L1311

This function searches for complete matches of " :", not ' ' __or__ ':' as stated.

As a result, pairs are not extracted, and shim libs are not parsed.

**Comment by krnlyng** [comment]
another issue is that the LD_SHIM_LIBS variable is not parsed in android_linker_init, that should be included in the fix.

**Comment by mlehtima** [comment]
These issues should be fixed once https://github.com/libhybris/libhybris/pull/377 is merged.
